### ISSUE TITLE
Error running az aks get version

### ISSUE THREAD

**Issue by ranand12** [Azure/azure-cli-extensions#2477]
## Describe the bug

**Command Name**
`az aks get-versions
Extension Name: aks-preview. Version: 0.4.47.`

**Errors:**
```
APIVersion 2020-04-01 is not available
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 215, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 654, in execute
    raise ex
cli/core/commands/__init__.py, ln 718, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py, ln 219, in container_services
    raise NotImplementedError("APIVersion {} is not available".format(api_version))
NotImplementedError: APIVersion 2020-04-01 is not available
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks get-versions -l {}`

## Environment Summary
```
Linux-4.15.0-1095-azure-x86_64-with-debian-stretch-sid (Cloud Shell)
Python 3.6.10
Installer: DEB

azure-cli 2.12.0 *

Extensions:
aks-preview 0.4.47
ai-examples 0.2.3

```

**Comment by yungezz** [comment]
routing to appropriate team

**Comment by RakeshMohanMSFT** [comment]
This is fixed in the latest version of azure cli [2.38.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
